t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -93,49 +93,54 @@ my $auth_header_exact
-        });
+            $c->res->headers->www_authenticate($www_authenticate);    # uncoverable statement
+            ;    # uncoverable statement
+            return $c->render(status => 401, text => 'login');    # uncoverable statement
+        });    # uncoverable statement
 
     $mock->routes->get(
-        '/build/ProjTestingSignature/_result' => sub {
-            my $c = shift;
+        '/build/ProjTestingSignature/_result' => sub {    # uncoverable statement
+            my $c = shift;    # uncoverable statement
 
-            if ($c->req->headers->authorization && $auth_header_exact eq $c->req->headers->authorization) {
-                return $c->render(status => 200, text => $fake_response_by_project{Proj1});
-            }
+            my $client_auth_header = $c->req->headers->authorization // '';    # uncoverable statement
+
+            if ($auth_header_exact eq $client_auth_header) {    # uncoverable statement
+
+                return $c->render(status => 200, text => $fake_response_by_project{Proj1});    # uncoverable statement
+            }    # uncoverable statement
+
+            $c->res->headers->www_authenticate($www_authenticate);    # uncoverable statement
+            return $c->render(status => 401, text => 'login');    # uncoverable statement
+        });    # uncoverable statement
 
-            $c->res->headers->www_authenticate(qq(Signature realm="Use your developer account",headers="(created)"));
-            return $c->render(status => 401, text => 'login');
-        });
-
-    for my $project (sort keys %fake_response_by_project) {
-        $mock->routes->get(
-            "/build/$project/_result" => sub {
-                my $c = shift;
-                my $pkg = $c->param('package');
-                return $c->render(status => 404) if !$pkg and $project ne 'Proj1';
-                return $c->render(status => 200, text => $fake_response_by_project{$project});
-            });
+    for my $project (sort keys %fake_response_by_project) {    # uncoverable statement
+        $mock->routes->get(    # uncoverable statement
+            "/build/$project/_result" => sub {    # uncoverable statement
+                my $c = shift;    # uncoverable statement
+                my $pkg = $c->param('package');    # uncoverable statement
+                return $c->render(status => 404) if !$pkg and $project ne 'Proj1';    # uncoverable statement
+                return $c->render(status => 200, text => $fake_response_by_project{$project});   # uncoverable statement
+            });    # uncoverable statement
     }
-    my $daemon = Mojo::Server::Daemon->new(app => $mock, listen => [$host]);
-    $daemon->run;
-    note 'Fake API server stopped';
-    _exit(0);
+    my $daemon = Mojo::Server::Daemon->new(app => $mock, listen => [$host]);    # uncoverable statement
+    $daemon->run;    # uncoverable statement
+    note 'Fake API server stopped';    # uncoverable statement
+    _exit(0);    # uncoverable statement
```

Comment:
So the following doesn't work?
```suggestion
    Devel::Cover::report() if Devel::Cover->can('report');
    _exit(0);    # uncoverable statement
```
Although I guess all these `# uncoverable statement` comments aren't *that* bad.

Reply:
It didn't affect the coverage report at all.